Matrix.py

```diff
@@ -0,0 +1,123 @@
+class Matrix:
+    def __init__(self, x):
+        if type(x) is tuple:
+            for tup in x:
+                if len(tup) == len(x) and type(tup) is tuple:
+                    self.x = x
```

Comment:
Why is this line inside a for loop? It should only happen once after you've check everything

Reply:
i don't have another idea except of counter...